vip-config/vip-config.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Hi there, VIP dev!
+ *
+ * vip-config.php is where you put things you'd usually put in wp-config.php. Don't worry about database settings
+ * and such, we've taken care of that for you. This is a good place to define a constant or something of that
+ * nature. However, consider using environment variables for anything sensitive or environment-specific:
+ *
+ * @see https://docs.wpvip.com/how-tos/manage-environment-variables/
+ *
+ * WARNING: This file is loaded very early (immediately after `wp-config.php`), which means that most WordPress APIs,
+ *   classes, and functions are not available. The code below should be limited to pure PHP.
+ *
+ * @see https://docs.wpvip.com/technical-references/vip-codebase/vip-config-directory/
+ *
+ * Happy coding!
+ *
+ * - The WordPress VIP Team
+ **/
+
+/**
+ * Redirect non www to www.
+ *
+ * @see https://docs.wpvip.com/redirects/domain-redirects-in-vip-config-php/
+ */
+if ( isset( $_SERVER['HTTP_HOST'] ) && isset( $_SERVER['REQUEST_URI'] ) ) {
+	$http_host   = $_SERVER['HTTP_HOST']; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+	$request_uri = $_SERVER['REQUEST_URI']; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+}
+
+/**
+ * Set a high default limit to avoid too many revisions from polluting the database.
+ *
+ * @see https://docs.wpvip.com/technical-references/vip-platform/post-revisions/
+ *
+ * Posts with high revisions can result in fatal errors or have performance issues.
+ *
+ * Feel free to adjust this depending on your use cases.
+ */
+if ( ! defined( 'WP_POST_REVISIONS' ) ) {
+	define( 'WP_POST_REVISIONS', 100 );
+}
```

Comment:
I wonder if this should be moved to Alleyvate. maybe @kevinfodness can weigh in on this? I assume we would want this same approach to employed on Pantheon sites as well as VIP sites.

Reply:
It comes from https://github.com/Automattic/vip-go-skeleton/blob/production/vip-config/vip-config.php

We might want to add it to Alleyvate for non-VIP sites, yes, but for our purposes here we're cloning what VIP is providing us with in their skeleton.

Another possibility for how to tackle this would be to actually fetch the latest and greatest from the VIP skeleton repo rather than maintaining a copy here. If installing VIP, then clone skeleton, copy relevant files, etc.

Reply:
i think we should make a separate issue for the automating the download of the VIP Config file.